src/main/java/model/Cars.java

```diff
@@ -0,0 +1,62 @@
+package model;
+
+import util.RandomNumberGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(String carNameInput) {
```

Comment:
```suggestion
    public Cars(String carNameInput) {
        this(createCars(carNameInput);
    }
    
    public Cars(List<Car> cars) {
       this.cars = cars;
    }
```
- 생성자는 기본으로 객체의 필드와 같은 타입(List<Car>)을 가지는 주 생성자를 기본으로 추가
- 다른 타입의 부 생성자는 위와 같이 주 생성자를 호출해 필드 값을 초기화

생성자는 위와 같은 기준을 가지고 구현해볼 것을 추천